example.env

```diff
@@ -1,5 +1,4 @@
 NEXMO_API_KEY=
 NEXMO_API_SECRET=
-NEXMO_DEBUG=true
-PROVISIONED_NUMBERS=[{"country":"2_CHAR_CODE","msisdn":"NUMBER"}]
-SMS_WEBHOOK_URL=
+VIRTUAL_NUMBER_A=
+VIRTUAL_NUMBER_B=
```

Comment:
Why do we need two virtual numbers here? We can link two numbers to a single number and then distribute the message based on the `from` field

Reply:
Great idea! Refactored code to achieve this